src/more-infos/more-info-hvac.js

```diff
@@ -0,0 +1,238 @@
+import hass from '../util/home-assistant-js-instance';
+
+import Polymer from '../polymer';
+import attributeClassNames from '../util/attribute-class-names';
+
+const { serviceActions } = hass;
+const ATTRIBUTE_CLASSES = ['away_mode', 'aux_heat'];
+
+export default new Polymer({
+  is: 'more-info-hvac',
+
+  properties: {
+    stateObj: {
+      type: Object,
+      observer: 'stateObjChanged',
+    },
+
+    tempMin: {
+      type: Number,
+    },
+
+    tempMax: {
+      type: Number,
+    },
+
+    humMin: {
+      type: Number,
+    },
+
+    humMax: {
+      type: Number,
+    },
+
+    targetTemperatureSliderValue: {
+      type: Number,
+    },
+
+    targetHumiditySliderValue: {
+      type: Number,
+    },
+
+    operationMode: {
+      type: String,
+      value: '',
+    },
+
+    operationIndex: {
+      type: Number,
+      value: 0,
+      observer: 'handleOperationmodeChanged',
+    },
+
+    fanMode: {
+      type: String,
+      value: '',
+    },
+
+    fanIndex: {
+      type: Number,
+      value: 0,
+      observer: 'handleFanmodeChanged',
+    },
+
+    swingMode: {
+      type: String,
+      value: '',
+    },
+
+    swingIndex: {
+      type: Number,
+      value: 0,
+      observer: 'handleSwingmodeChanged',
+    },
+
+
+    awayToggleChecked: {
+      type: Boolean,
+    },
+
+    auxToggleChecked: {
+      type: Boolean,
+    },
+  },
+
+  stateObjChanged(newVal) {
+    this.targetTemperatureSliderValue = newVal.attributes.temperature;
+    this.targetHumiditySliderValue = newVal.attributes.humidity;
+    this.fanMode = newVal.attributes.fan;
+    this.swingMode = newVal.attributes.swing_mode;
+    this.operationMode = newVal.attributes.operation_mode;
+    this.awayToggleChecked = newVal.attributes.away_mode === 'on';
+    this.auxheatToggleChecked = newVal.attributes.aux_heat === 'on';
+
+    this.tempMin = newVal.attributes.min_temp;
+    this.tempMax = newVal.attributes.max_temp;
+    this.humMin = newVal.attributes.min_humidity;
+    this.humMax = newVal.attributes.max_humidity;
+
+    if (newVal.attributes.fan_list !== null) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/16622752/15099469/23ce36fc-1556-11e6-8cd2-84dcc0718cbb.png)
This is the persisiting problem:
The first more-info is displaying like it should everything is highlighted and displayed where it should.
However, when changing to a different entity more-info, you can see that in the list, the correct item is highlighted, but the displayed selection is wrong.